### PR TITLE
Support multi filters in one column

### PR DESCRIPTION
**Pull request by tkuchiki**
- support multi filters in one column
- add `-or` option

# Examples

## And

```console
$ ./csviewer -p _example/example.csv -f "id > 2 && id <= 10"
+----+------+-----------------+-----------+--------+
| ID | NAME |      MAIL       |   PHONE   | ADRESS |
+----+------+-----------------+-----------+--------+
|  3 | c    |                 |           |  22222 |
|  5 | d    | ddd@fuga.hgoe   | 123456789 |        |
| 10 | e    | eeeee@fuga.hgoe |    654321 |        |
+----+------+-----------------+-----------+--------+
```

```console
$ ./csviewer -p _example/example.csv -f "id > 2" -f "id <= 10"
+----+------+-----------------+-----------+--------+
| ID | NAME |      MAIL       |   PHONE   | ADRESS |
+----+------+-----------------+-----------+--------+
|  3 | c    |                 |           |  22222 |
|  5 | d    | ddd@fuga.hgoe   | 123456789 |        |
| 10 | e    | eeeee@fuga.hgoe |    654321 |        |
+----+------+-----------------+-----------+--------+
```

## Or

```console
$ ./csviewer -p _example/example.csv -f "name == 'c'" -f "name == 'd'" -or
+----+------+---------------+-----------+--------+
| ID | NAME |     MAIL      |   PHONE   | ADRESS |
+----+------+---------------+-----------+--------+
|  3 | c    |               |           |  22222 |
|  5 | d    | ddd@fuga.hgoe | 123456789 |        |
+----+------+---------------+-----------+--------+
```

```console
$ ./csviewer -p _example/example.csv -f "name == 'c' || name == 'd'"
+----+------+---------------+-----------+--------+
| ID | NAME |     MAIL      |   PHONE   | ADRESS |
+----+------+---------------+-----------+--------+
|  3 | c    |               |           |  22222 |
|  5 | d    | ddd@fuga.hgoe | 123456789 |        |
+----+------+---------------+-----------+--------+
```

### CODE REVIEW

File: csviewer.go [hunk 1]
```diff
@@ -1,31 +1,33 @@
...
 
 import (
+	"fmt"
 	"log"
 	"os"
 	"sort"
 	"strconv"
 	"strings"
 
+	"github.com/Knetic/govaluate"
```
Comment (Konboi) [inline csviewer.go:11]: Please Add Gopkg.toml and update  Gopkg.lock

File: csviewer.go [hunk 2]
```diff
@@ -245,15 +204,32 @@ func (v *Csviewer) Print(opt *sortOption) {
...
+	op := " && "
+	if v.isFiltersOr {
+		op = " || "
+	}
+
+	expression, err := govaluate.NewEvaluableExpression(strings.Join(filters, op))
+	if err != nil {
+		log.Fatal(err)
+	}
+	result, err2 := expression.Evaluate(values)
```
Comment (Konboi) [inline csviewer.go]: Please be err instead of err2

```go
result, err := expression.Evaluate(values)
```

**Review by Konboi — COMMENTED** [review]
Thank you, PR
I made some comments

**Comment by tkuchiki** [comment]
@Konboi Thank you for your review.
Fix it!

**Review by Konboi — APPROVED** [review]
Thank You!!

**Pull request merged**